uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -711,10 +711,6 @@ Determines whether numerical value falls within the range described by range
 string. Range string is a single range unit which follows the syntax
```

Comment:
Has the `.md` been regenerated?

Reply:
@nicolasbrossard oops. I just regenerated it. Thanks!